src/events/maxisonline.py

```diff
@@ -114,8 +114,9 @@ async def on_presence_update(self, before: discord.Member, after: discord.Member
             # Get Max's info only
             if before.id == self.max_member.id:
                 if before.guild.id == 730196305124655176:
-                    if (before.status != discord.Status.online and after.status == discord.Status.online) \
+                    if (before.is_on_mobile() and after.is_on_mobile()) \
```

Comment:
Not what I said :)

If the user stays on their phone that doesn't qualify a post does it?

Reply:
mistype
very loud music here